src/Stripe.net/Entities/Source.cs

```diff
@@ -15,6 +16,7 @@ public class Source : StripeEntityWithId
     {
         public SourceType Type { get; set; }
 
+        public StripeAccount Account { get; set; }
```

Comment:
It seems easier to use `StripeAccount` here even though the API only returns this:
```
{
  id: "acct_XXXXX",
  object: "account"
}
```

Reply:
👍